f/assignGear/f_assignGear_aaf_standard.sqf

```diff
@@ -228,7 +232,7 @@ switch (_typeofUnit) do
 		_unit addmagazines [_smokegrenade, 3];
 		_unit addmagazines [_carbinemag, 2];
 		_unit addmagazines [_carbinemag_tr, 2];
-		_unit addmagazines [_MATmag1, 2];
+		_unit addmagazines [_MATmag1, 4];
```

Comment:
Shouldn't this be an increase in _carbinemag rather than an increase in _MATmag1?